app/etl/ga/service.rb

```diff
@@ -14,8 +14,29 @@ def find_in_batches(date:, batch_size: 10_000)
       .each_slice(batch_size) { |slice| yield slice }
   end
 
+  def find_user_feedback_in_batches(date:, batch_size: 10_000)
```

Comment:
This class is doing too much. What about extracting a class to deal with user feedback events?

Reply:
Please, notice the `services` folder has changed. Please, split the specs as well.